care/facility/api/viewsets/patient.py

```diff
@@ -630,8 +633,31 @@ def perform_create(self, serializer):
             raise ValidationError(
                 {"patient": "Only active patients data can be updated"}
             )
-        return serializer.save(
+
+        message = {
+            "type": "MESSAGE",
+            "facility_id": str(patient.facility.external_id),
+            "patient_id": str(patient.external_id),
+            "status": "updated",
+        }
+
+        facility_users = FacilityUser.objects.filter(facility_id=patient.facility.id)
+        for facility_user in facility_users:
+            if facility_user.user.id != self.request.user.id:
+                send_webpush(username=facility_user.user, message=json.dumps(message))
```

Comment:
@sainak is this call deterministic, if not can we move it to the notification generator, it send the webpush from the worker.